src/style.css

```diff
@@ -1,3 +1,63 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  justify-content: space-between;
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.nav {
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  font-size: 15px;
+  display: block;
+}
+
+.nav__item {
+  margin: auto 20px;
+}
+
+a.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+:hover {
```

Comment:
Using the universal `:hover` selector applies the style to all elements on hover, which may not be the intended behavior. Consider specifying a more targeted selector, such as `.nav__link:hover`.